Dockerfile

```diff
@@ -128,24 +110,52 @@ RUN cd /linux-kernel && \
 # Prepare the ISO directory with the kernel
 RUN cp -v /linux-kernel/arch/x86_64/boot/bzImage /tmp/iso/boot/vmlinuz64
 
+ENV TCL_REPO_BASE   http://tinycorelinux.net/7.x/x86_64
+# Note that the ncurses is here explicitly so that top continues to work
+ENV TCZ_DEPS        iptables \
+                    iproute2 \
+                    openssh openssl \
+                    tar \
+                    gcc_libs \
+                    ncurses \
+                    acpid \
+                    xz liblzma \
+                    git expat2 libgpg-error libgcrypt libssh2 \
+                    nfs-utils tcp_wrappers portmap rpcbind libtirpc \
+                    rsync attr acl \
+                    curl ntpclient \
+                    procps glib2 libtirpc libffi fuse pcre \
+                    udev-lib udev-extra \
+                    liblvm2 \
+                    parted
+
 # Download the rootfs, don't unpack it though:
 RUN curl -fL -o /tcl_rootfs.gz $TCL_REPO_BASE/release/distribution_files/rootfs64.gz
 
 # Install the TCZ dependencies
-RUN for dep in $TCZ_DEPS; do \
-    echo "Download $TCL_REPO_BASE/tcz/$dep.tcz" &&\
-        curl -fL -o /tmp/$dep.tcz $TCL_REPO_BASE/tcz/$dep.tcz && \
-        unsquashfs -f -d $ROOTFS /tmp/$dep.tcz && \
-        rm -f /tmp/$dep.tcz ;\
+RUN set -ex && \
```

Comment:
Adding `set -e` here helped me find a number of the `TCZ_DEPS` that don't exist anymore (and don't seem to be necessary either), since they 404 and fail now.  I think `libiconv` and `libidn` were the two most notable examples.